includes/utils.php

```diff
@@ -448,6 +462,8 @@ function prepare_meta( $post_id ) {
 		}
 	}
 
+	$prepared_meta = apply_filters( 'dt_prepared_meta', $prepared_meta, $post_id );
```

Comment:
I've added a suggestion for the docblock.

I'm wondering if the check for excluded meta data should be done after this filter is applied to ensure against distributing disallowed meta (it will get blocked on save anyway).

Would this be problematic in your use case?

```suggestion
	/**
	 * Filter prepared meta for consumption.
	 *
	 * Modify meta data before it is sent for consumption by a distributed
	 * post. The prepared meta data should not include any excluded meta.
	 * see `excluded_meta()`.
	 *
	 * @since 2.0.0
	 * @hook dt_prepared_meta
	 *
	 * @param {array} $prepared_meta Prepared meta.
	 * @param {int}   $post_id      Post ID.
	 *
	 * @return {array} Prepared meta.
	 */
	$prepared_meta = apply_filters( 'dt_prepared_meta', $prepared_meta, $post_id );
```

Reply:
@peterwilsoncc Not at all, that makes sense to me.